### PR TITLE
Add Resolver for Streamcherry

### PR DESCRIPTION
**Pull request by csong95**
no comment

### CODE REVIEW

**Comment by SpajicM** [comment]
Please use BaseResolver.

**Comment by SpajicM** [comment]
Ok, just make sure to squash merge so we don't have 4 commits that should have been 1 :)

**Comment by ddivad** [comment]
@csong95 Thanks for the PR. 

If you are going to implement the BasResolver for this resolver, you should also add the `supportsUri` function and then remove the `else if` from `resolve.js` as it will be handled by the for loop for BasResolver classes at the top of the file.

If you want an example, you can look at the `TikiWiki`resolver.

File: src/scrapers/resolvers/resolveHtml.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const rp = require('request-promise');
 const resolvers = {
     Openload: require('./Openload').OpenloadHtml,
     Streamango: require('./Streamango').StreamangoHtml,
+    //Streamcherry: require('./Streamcherry').StreamcherryHtml,
```
Comment (j0j00) [inline src/scrapers/resolvers/resolveHtml.js]: If the resolver isn't IP locked, then the changes in this file should probably be removed?

Comment (kevincosmos) [reply src/scrapers/resolvers/resolveHtml.js]: @csong95 are you able to update this?

Comment (kevincosmos) [reply src/scrapers/resolvers/resolveHtml.js]: @ddivad @SpajicM what is the plan for this going forward based on the discussion yesterday?

**Comment by Teekzilla** [comment]
Closing this as there doesn't seem to be any activity. If @csong95 wants to continue, we can reopen it, otherwise I might take a look at this after getting a better understanding of `_BaseResolver` and JW Player in general.

**Pull request closed without merging**